cross-images/aarch64-unknown-linux-gnu.dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM rustembedded/cross:aarch64-unknown-linux-gnu
+
+COPY libpq.sh /
+RUN bash /libpq.sh aarch64-linux-gnu
+
+RUN apt-get update -y && \
+    apt-get install -y libpq-dev
```

Comment:
```suggestion
    apt-get install -y libpq-dev && \
    rm -rf /var/lib/apt/lists/*
```

Reply:
Note that this is actually not necessary here given the purpose of this image, but it's just a good habit.